.github/workflows/base-wrapper.yml

```diff
@@ -33,10 +33,6 @@ on:
         type: string
         description: 'Mandrel packaging repository to be used'
         default: "graalvm/mandrel-packaging"
-      distribution:
-        type: string
-        description: 'Distribution to build, mandrel or graalvm'
-        default: "mandrel"
```

Comment:
Unfortunately we can't remove this since we need to be able to test both Graal and Mandrel builds

Reply:
Fair enough. How about we combine some of the inputs? For example we could combine `build-from-source` with `distribution` like so:

```
inputs:
      buildType:
        description: 'Build from source instead of using a release for graal or mandrel distribution"
        required: true
        default: 'build-from-source-mandrel' 
        type: choice
        options:
        - build-from-source-mandrel
        - build-from-source-graalvm
        - release-mandrel
        - release-graalvm
```

Reply:
Sure, that's what we did with the JDK in the past.

Note that `choice` doesn't work well with wrappers IIRC so you might need to use a `string` instead.

Regarding the options, I suggest we keep them shorter, e.g. `mandrel`, `graalvm`, `mandrel-release` and `graalvm-release`